lgc/patch/PatchBufferOp.cpp

```diff
@@ -1645,6 +1657,53 @@ Instruction *PatchBufferOp::makeLoop(Value *const loopStart, Value *const loopEn
   return loopCounter;
 }
 
+// =====================================================================================================================
+// Provide well-defined behavior for accessing null descriptor.
+//
+// @param checkInst : The buffer descriptro user.
+void PatchBufferOp::processDescriptorUser(Value *checkInst) {
+  ExtractElementInst *extractInst = nullptr;
+  unsigned descIdx = 0;
+  Instruction *insertPos = nullptr;
+  if (isa<ExtractElementInst>(checkInst)) {
```

Comment:
You do not need to use isa followed by dyn_cast (or cast). Just use dyn_cast and check the result for nullptr.

Reply:
Thanks. I will fix it.